### ISSUE TITLE
pyperclip work with interactive shell but not in script

### ISSUE THREAD

**Issue by Saltperfect** [asweigart/pyperclip#118]
#! /usr/bin/env python3
		# pw.py - An insecure password locker program.

		PASSWORDS = {'email': 'F7minlBDDuvMJuxESSKHFhTxFtjVB6',
		             'blog': 'VmALvQyKAxiVH5G8v01if1MLZF3sdt',
		             'luggage': '12345'}

		import sys, pyperclip
		if len(sys.argv) < 2:
		    print('Usage: py pw.py [account] - copy account password')
		    sys.exit()

		account = sys.argv[1]   # first command line arg is the account name

		if account in PASSWORDS:
		    pyperclip.copy(PASSWORDS[account])
		    print('Password for ' + account + ' copied to clipboard.')
		else:
		    print('There is no account named ' + account)
I have used the pyperclip in terminal and it worked perfectly but when I wrote a script to use the module, the code executes completely and prints in the terminal whatever it is supposed to print but I never get data, in this case password on my clipboard. Moreover If I have already copied something on my clipboard that also gets erased from my clipboard. 
I am using python 3.6.0 on anaconda on ubuntu 16.04.

**Comment by modax** [comment]
@Saltperfect I believe you're suffering from https://github.com/asweigart/pyperclip/issues/116

**Comment by SamMatzko** [comment]
@modax No, not quite. This issue is different from #116 in that #116 deals with lag; _this_ issue is about it _not happening at all_.

I'm having this same problem on my Ubuntu 20.04, where running `copy()` in the Python shell works, but running it in a script doesn't.

For example, this simple script doesn't work. When I try to paste after running it, nothing happens:

```python
import pyperclip
pyperclip.copy("asldkfjaklejlkawef;alsdkfj;awle 'a")
```

But these same lines as shell commands _do_ work. When I paste after running them, the text is pasted:

```python
Python 3.8.10 (default, Mar 15 2022, 12:22:08) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyperclip
>>> pyperclip.copy("asldkfjaklejlkawef;alsdkfj;awle 'a")
```

I would be nice to have a fix for this, but I'm unsure what's causing the problem. Perhaps one of you `pyperclip` developers could look into it?

**Comment by RojemVega** [comment]
I know this is an inactive tread but i solved my problem in ubuntu 23 adding this line before copy:

import pyperclip as pc
import platform
if platform.system() == "Linux": pc.set_clipboard("xclip")
pc.copy("test string")

you will need to install xclip in your Ubuntu before.
